sonar-ruby-plugin/src/main/java/org/sonarsource/ruby/converter/RubyVisitor.java

```diff
@@ -143,7 +198,7 @@ private NativeTree createNativeTree(AstNode node, List<Object> children) {
     return new NativeTreeImpl(metaData(node), new RubyNativeKind(node.type()), nonNullChildren);
   }
 
-  private static Stream<Tree> treeForChild(@Nullable Object child, TreeMetaData treeMetaData) {
```

Comment:
why `@Nullable` is removed? I think it should be there

Reply:
Oops. Mistake when rebasing on master. Fixed it :+1: